app/src/components/DuckDBShell/DuckDBShell.tsx

```diff
@@ -0,0 +1,210 @@
+import { useState } from 'react'
+import { getDB } from '../../db/getDB'
+
+const MAX_ROWS = 1000
+const MAX_HISTORY = 20
+
+export function DuckDBShell() {
+    const [query, setQuery] = useState(
+        'SELECT 42 AS answer;\n-- tips: use `show tables` to… show tables'
+    )
+    const [results, setResults] = useState<
+        { columns: string[]; rows: string[][]; totalRows: number } | undefined
+    >(undefined)
```

Comment:
```suggestion
    const [results, setResults] = useState<
        { columns: string[]; rows: string[][]; totalRows: number } | undefined
    >()
```
no need to specify undefined value, as it's by default undefined